schedule/yast/maintenance/sap_autoyast_create_hdd_textmode_netweaver_sle12.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: sap_autoyast_create_hdd_gnome_netweaver_sle12
+description: '|
```

Comment:
could you edit the existing file for gnome if it is owned by us and we don't use it anywhere else?
if we still need a duplicated file with few adjustment for textmode, could you please put those adjustment out and share the same file (or file another ticket also could work)?

Reply:
We need keep this gnome file, as it needed by x86_64 platform.

Reply:
I will file another ticket to adjust those two yaml files.